app/views/admin/editions/show/_main.html.erb

```diff
@@ -91,6 +91,17 @@
           <% end %>
         <% end %>
       <% end %>
+    <% elsif !@edition.publicly_visible? && !@edition.change_note_section_completed? %>
+      <%= render "govuk_publishing_components/components/heading", {
```

Comment:
i know this is a duplication, but i'm going to add a follow up PR which moves the preview logic into a view component as there's tonnes of untested logic here right now.